test/parallel/test-timers-unrefed-in-callback.js

```diff
@@ -47,15 +46,11 @@ function Test2() {
   // server only for maintaining event loop
   const server = net.createServer().listen(0);
 
-  const timer2 = setInterval(() => {
+  const timer2 = setInterval(common.mustCall(() => {
     timer2.unref();
     if (counter2++ === 3)
       server.close();
-  }, 1);
+  }, 4), 1);
```

Comment:
I'm not sure this change is correct, especially given the comment block preceding this second test.